src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -24,9 +24,10 @@ limitations under the License.
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <description>This cluster provides an interface for launching content on a media player device such as a TV or Speaker.</description>
+    <globalAttribure side="either" code="0xFFFD" value="2"/>
```

Comment:
So this is fundamentally "proposed for 1.3", right?

Reply:
You are right and I did not want that. Moved back to 1.2.